src/v/serde/serde.h

```diff
@@ -54,12 +54,6 @@ inline constexpr bool serde_is_enum_v =
   std::is_enum_v<T>;
 #endif
 
-#if defined(SERDE_TEST)
```

Comment:
To make 9d7a877c65fb7d1d153d4f6372ff40f7789e0e25 compile correctly you'll need to add `serde_size_t.h` as an include somewhere right?

Reply:
Added `#include "serde_size_t.h"`.